compiler/rustc_codegen_llvm/src/llvm/ffi.rs

```diff
@@ -1175,6 +1183,7 @@ extern "C" {
     pub fn LLVMRustCreateUWTableAttr(C: &Context, async_: bool) -> &Attribute;
     pub fn LLVMRustCreateAllocSizeAttr(C: &Context, size_arg: u32) -> &Attribute;
     pub fn LLVMRustCreateAllocKindAttr(C: &Context, size_arg: u64) -> &Attribute;
+    pub fn LLVMRustCreatMemoryEffectsAttr(C: &Context, effects: MemoryEffects) -> &Attribute;
```

Comment:
Typo: s/Creat/Create/

(presumably elsewhere, since this compiles)

Reply:
Fixed.